src/python/grpcio_tests/tests/testing/_server_application.py

```diff
@@ -37,7 +37,7 @@ def UnStre(self, request, context):
         if _application_common.UNARY_STREAM_REQUEST != request:
             context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
             context.set_details('Something is wrong with your request!')
-        return
+            return
```

Comment:
Yes, it does change the behavior of the test. My sense is the new behavior is the intended one, i.e. the test was buggy before. Don't you agree?

Reply:
Not at all. Yielding zero messages in all invocations was always the intended behavior of this method.

Reply:
Okay; `yield` with a value was misleading.

Reply:
How could we face others without type-correct code (however unreachable)? How could we face ourselves?

Reply:
I don't agree with that characterization. If we agree `return; yield` to be an acceptable idiom (something that pylint folks obviously disagree, but that's another argument), it is much preferable to recognize that two statements together as something special with the return type of "empty sequence of anything", which would be type-correct. I strongly disagree that returning a random value to satisfy the static analyzer is a good idea.  It at least needs to be a known sentinel, not `.Strange()`, which makes the developer think it was intended (in fact I just made that mistake precisely because I thought it was supposed to be a meaningful value).